metadata.yaml

```
@@ -9,3 +9,11 @@ summary: A MongoDB operator charm
 series:
   - focal
 
+storage:
+  mongodb:
+    type: filesystem
+    location: /data/db
+
+peers:
+  mongodb:
+    interface: mongodb
```

```suggestion
    interface: mongodb-replicas
```

What are you going to name the interface with other applications? Might just be worth being explicit here.